README.md

```diff
@@ -178,13 +178,14 @@ Run the beacon node
 
 ```shell
 bazel run //beacon-chain --define=ssz=minimal -- \
---datadir="./$TESTNET_NAME/nodes/prysm0/beacondata" \
---min-sync-peers=0 \
---http-web3provider=http://127.0.0.1:8545 \
---bootstrap-node= \
---chain-config-file="./$TESTNET_NAME/public/eth2_config.yaml" \
---genesis-state="./$TESTNET_NAME/public/genesis.ssz" \
---minimal-config
+ --datadir="./$TESTNET_NAME/nodes/prysm0/beacondata" \
+ --min-sync-peers=0 \
+ --http-web3provider=http://127.0.0.1:8545 \
+ --bootstrap-node= \
```

Comment:
Is this option supposed to be empty?

Reply:
Yes, added a small comment below the command